sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -7993,7 +8012,7 @@
                     {
                       "components": [
                         {
-                          "html": "<p style=\"text-align:right;\">15MB file limit each</p>\n",
+                          "html": "<p style=\"text-align:right;\">4MB file limit each</p>\n",
```

Comment:
The form definitions does not seem to be updated on top of latest code from main. This is reverting the change implemented in #2870